include/powerloader/curl.hpp

```diff
@@ -35,17 +36,25 @@ namespace powerloader
         bool m_serious;
     };
 
-    struct POWERLOADER_API Response
+    struct POWERLOADER_API BaseResponse
     {
-        std::map<std::string, std::string> header;
-        mutable std::stringstream content;
+        std::map<std::string, std::string> headers;
 
-        curl_off_t avg_speed;
+        curl_off_t average_speed;
         curl_off_t downloaded_size;
         long http_status;
```

Comment:
Theres members need to be set to a default value, otherwise for example calling `ok()` is undefined behavior.

Reply:
Done